core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1035,6 +1035,18 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_RPC_READ_COUNT =
+      new Builder("Worker.RpcReadCount")
+          .setDescription("The number of active read-RPCs managed by this worker")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(true)
+          .build();
+  public static final MetricKey WORKER_RPC_WRITE_COUNT =
+      new Builder("Worker.RpcWriteCount")
+          .setDescription("The number of active write-RPCs managed by this worker")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(true)
+          .build();
```

Comment:
Any suggestions on how to aggregate these two into a single `WORKER_RPC_COUNT`?

Also I assume we'd want these metrics to be cluster aggregated?

Reply:
the only way i know is to have another metric gauge which is the sum of these two counter values.

Reply:
Note-to-self; use `counterWithTag` for cluster-aggregated `Counter`s

@yuzhu Any suggestions on what parent class should own the gauge?

Reply:
I added `counterWithTag` and tested it but wasn't able to get the master metric sink (`:19999/metrics/json`) to show `Cluster.RpcReadCount`). Notably the `WORKER_BYTES_READ_REMOTE` metric `Cluster.BytesReadRemote` _did_ get incremented as a result of the test, so I'm not sure why the RPC counter didn't show up there too.

Reply:
Fixed in latest commit: https://github.com/Alluxio/alluxio/pull/13481/commits/fe321bd9e7648e0c4a06db88d90ad42b9dacbe3b